web/backend/src/cli.ts

```diff
@@ -83,6 +83,17 @@ program
       const scipers: Array<string> = data.split('\n');
       const policies = [];
       for (let i = 0; i < scipers.length; i += 1) {
+        const sciper: number = Number(scipers[i]);
+        if (Number.isNaN(sciper)) {
+          console.error(`SCIPER on line ${i + 1} is not a number - exiting!`);
```

Comment:
This must throw an error:

https://www.npmjs.com/package/commander#custom-option-processing

So if you use it in a script, you get an actual different exit code.

Same thing for the sciper-number comparison.

Reply:
Ok. now I'm raising an exception instead of printing and returning.

Reply:
Sorry, one more...

Reply:
done